gradle.properties

```diff
@@ -17,12 +17,6 @@ org.gradle.jvmargs=-Xmx1g -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryEr
 org.gradle.caching=true
 org.gradle.parallel=true
 
-# Use this flag so that when compiling against project dependencies
-# the jar is created and used instead of using src/main/classes.
-#
-# See https://docs.gradle.org/current/userguide/java_library_plugin.html#sub:java_library_known_issues_windows_performance
-systemProp.org.gradle.java.compile-classpath-packaging=true
-
```

Comment:
❓ Why is this no longer necessary?

Reply:
Because @bjhargrave cleverly figured out how to solve that inside the bnd gradle plugin :)